includes/Modules/Analytics.php

```diff
@@ -1094,6 +1062,48 @@ function ( $order_def ) {
 		throw new Invalid_Datapoint_Exception();
 	}
 
+	/**
+	 * Parses orderby value of the data request.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Data_Request $data Data request object.
+	 *
+	 * @return array An array of parsed orders.
+	 */
+	protected function parse_data_orderby( Data_Request $data ) {
+		$orderby = $data['orderby'];
+		if ( empty( $orderby ) || ! is_array( $orderby ) ) {
+			return array();
+		}
+
+		return array_filter(
+			array_map(
+				function ( $order_def ) {
+					$order_def = array_merge(
+						array(
+							'fieldName' => '',
+							'sortOrder' => 'DESCENDING',
+						),
+						(array) $order_def
+					);
+
+					if ( empty( $order_def['fieldName'] ) ) {
+						return null;
+					}
```

Comment:
I know @felixarntz suggested adding the default `sortOrder` back in the previous PR but if we're going to filter out invalid orderbys then I think we should require both to be provided rather than just the `fieldName` and leave out any defaults. The merging there is mainly to ensure the structure to prevent non-existing key warnings, but also because orderbys used to be actual objects due to the way batch requests worked previously.
I'm happy to defer to @felixarntz on this, but in my opinion we should avoid any default here and require both or filter it out.

Reply:
SGTM!

Reply:
Updated.